app/javascript/utils/statusUtils.js

```diff
@@ -1,13 +1,13 @@
-import { find } from 'lodash'
+import { find, map } from 'lodash'
 
 export const LEASE_UP_STATUS_OPTIONS = [
-  { value: 'Processing', label: 'Processing', style: 'is-processing', commentRequired: true },
-  { value: 'Withdrawn', label: 'Withdrawn', style: 'is-withdrawn' },
-  { value: 'Appealed', label: 'Appealed', style: 'is-appealed', commentRequired: true },
-  { value: 'Waitlisted', label: 'Waitlisted', style: 'is-waitlisted', commentRequired: true },
-  { value: 'Disqualified', label: 'Disqualified', style: 'is-disqualified', commentRequired: true },
-  { value: 'Approved', label: 'Approved', style: 'is-approved', commentRequired: true },
-  { value: 'Lease Signed', label: 'Lease Signed', style: 'is-leased' }
+  { value: 'Processing', label: 'Processing', style: 'is-processing', pillStyle: 'status-pill-processing', commentRequired: true },
+  { value: 'Withdrawn', label: 'Withdrawn', style: 'is-withdrawn', pillStyle: 'status-pill-withdrawn' },
+  { value: 'Appealed', label: 'Appealed', style: 'is-appealed', pillStyle: 'status-pill-appealed', commentRequired: true },
+  { value: 'Waitlisted', label: 'Waitlisted', style: 'is-waitlisted', pillStyle: 'status-pill-waitlisted', commentRequired: true },
+  { value: 'Disqualified', label: 'Disqualified', style: 'is-disqualified', pillStyle: 'status-pill-disqualified', commentRequired: true },
+  { value: 'Approved', label: 'Approved', style: 'is-approved', pillStyle: 'status-pill-approved', commentRequired: true },
```

Comment:
might be a little too late, but what's the value in having a separate style and pillStyle attribute?

Reply:
Well one's for a button and ones for a pill, the different classes apply different css attrs. I could calculate them from a single value like `style: 'approved'` and then in status pill use it like ``status-pill-${option.style}`` but I kind of feel like it's more searchable this way.

Can also update the `style` field to be `buttonStyle` to make it more clear

Reply:
i was thinking that maybe something like

`.status-pill .is-approved {}` could give the right style, but maybe that's a bad idea. I agree that having it be searchable is helpful